openshift/gabi.template.yaml

```diff
@@ -223,9 +223,9 @@ parameters:
 - name: RESOURCES
   value: '{"requests": {"memory": "128Mi", "cpu": "100m"}, "limits":{"memory": "256Mi"}}'
 - name: OAUTH_PROXY_IMAGE_NAME
-  value: quay.io/openshift/origin-oauth-proxy
+  value: registry.redhat.io/openshift4/ose-oauth-proxy-rhel9
```

Comment:
you will need to add a pull secret in the `ServiceAccount` to be able to pull this image.